docs/install.sh

```diff
@@ -26,44 +25,67 @@ fi
 
 echo "INFO: verifying cpu architecture compatibility"
 
-ARCH=$(uname -m)
-ARCH=$(echo "${ARCH,,}") # convert to lower case
+ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
 
 if [ "$ARCH" = x86_64 ]; then
     ARCH="amd64"
 fi
+if [ "$ARCH" = aarch64 ]; then
+    ARCH="arm64"
+fi
 
-if [ "$ARCH" != "amd64" ]; then
+if [[ "$ARCH" != "amd64" && "$ARCH" != "arm64" ]]; then
     echo "Error: ${ARCH} is not a supported architecture at this time."
     exit 1
 fi
 
-# ------------------------------------------------------------------------------
-# Determine Latest Release
-# ------------------------------------------------------------------------------
+TEMPDIR=$(mktemp -d)
```

Comment:
can we delete the tempdir when done with it? maybe something along the lines of `trap rm -rv "$TEMPDIR" EXIT` ?

Reply:
It's being done in lines 113-115. Not via a `trap` (which would be better) but it's there.

Reply:
Created an issue for this: #531